controllers/articlesController.js

```diff
@@ -56,4 +56,46 @@ export default class PostController {
       return res.status(500).send({ message: error, status: 500 });
     }
   }
+
+  /**
```

Comment:
Good description, please do the same in the rest of your files to keep consistency